README.md

```diff
@@ -201,6 +207,95 @@ const rule = {
 };
 ```
 
+### Includes
+
+The includes rule has three distinct behaviors, depending on whether the input is a string, an array, or an object.
+
+##### Input is a String
+
+When input is a string, the includes rule will check if the rule property in the config is a substring of the input.
+
+```yaml
+isSomeFeatureEnabled:
+  values:
+    - value: true
+      name: john
+    - value: false
+```
+
+```javascript
+const rule = {
+  type: 'includes',
+  input: input => input,
+  property: 'name',
+};
+...
+flipr.getValue(isSomeFeatureEnabled, 'john'); // true
```

Comment:
should `isSomeFeatureEnabled` be wrapped with single quotes?  same for the others